package.json

```diff
@@ -55,7 +55,8 @@
     "docs": "node scripts/generate-readme.js",
     "simple-integration-test": "jest --maxWorkers=5 simple-suite",
     "complex-integration-test": "jest --maxWorkers=5 integration",
-    "postinstall": "node ./scripts/postinstall.js"
+    "postinstall": "node ./scripts/postinstall.js",
+    "pre-release": "node ./scripts/pre-release.js"
```

Comment:
This could be `prepublish` or `prepack` (https://docs.npmjs.com/misc/scripts) so that it's automatic or should be documented in the `REALEASE-CHECKLIST.md` file.